www/components/SPA/SymbolPage.tsx

```diff
@@ -11,10 +11,28 @@ import ClassContents from "@/components/Sidebar/ClassContents";
 import ModuleContents from "@/components/Sidebar/ModuleContents";
 import ModuleList from "@/components/Sidebar/ModuleList";
 
+import * as docs from "@/lib/docs";
 import { withoutPrefix } from "@/lib/url";
 
+function resolveClass(mod: docs.Module, name: string): docs.Class | undefined {
+  const parts = withoutPrefix(mod.name, name).split(".");
+  let cls = mod.classes.find(
+    (c) => withoutPrefix(mod.name, c.name) === parts.shift()
```

Comment:
I come from the future, this code doesn't work. :)

Reply:
https://github.com/zsol/py.wtf/commit/51aa1ff305a0230461419a705168d95721bda358

Reply:
Ooooooh. Dammit, and I even had tests! And apparently even those were only for the first class. Dammit.